back/spec/acceptance/users_spec.rb

```diff
@@ -8,6 +8,9 @@
 
   before do
     header 'Content-Type', 'application/json'
+    settings = AppConfiguration.instance.settings
```

Comment:
I don't think this is related to your PR, right? I also don't see it on master, so probably should be removed